Update README.md

bytestream opened the pull request:
No `PUT` / `DELETE` end points are supported. I changed the unsupported section to include a `PUT` example.

Gaitholabi commented:
> No `PUT` / `DELETE` end points are supported. I changed the unsupported section to include a `PUT` example.

The last PR adds  two 'PUT' endpoints, so I'd change it to 'DELETE'

bytestream commented:
> 
> 
> > No `PUT` / `DELETE` end points are supported. I changed the unsupported section to include a `PUT` example.
> 
> The last PR adds two 'PUT' endpoints, so I'd change it to 'DELETE'

I've switched it from `put` to `delete`. If you could double check that's correct please 🤞

jshah4517 commented:
Suggest to close in favour of https://github.com/supportpal/api-client-php/pull/49
